src/usr/bin/gce-nic-naming

```diff
@@ -492,5 +508,20 @@ if [[ ! $TEST == 'test' ]]; then
       determine_index_ratios
   fi
 
-  generate_name "${DEVPATH}"
+  generated_name=$(generate_name "${DEVPATH}")
+  
+  # handle the 2nd load order 
```

Comment:
1. net
2. irdma
right? Can you say this explicitly? Assumptions about load order should be documented for anyone using a custom kernel.

Reply:
Done.